Documentation/ceph-upgrade.md

```diff
@@ -221,6 +221,16 @@ Replace the namespace names in the new resources:
 sed "s/ROOK_SYSTEM_NAMESPACE/$ROOK_SYSTEM_NAMESPACE/g" upgrade-from-v1.0-create.yaml > upgrade-from-v1.0-create.yaml.tmp
 sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-create.yaml.tmp > upgrade-from-v1.0-create.yaml
 rm -f upgrade-from-v1.0-create.yaml.tmp
+sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-apply.yaml
+```
+
+If you have a v1.0 cluster running with CSI drivers enabled, delete the rbac rules created for CSI
```

Comment:
Should *not* do this step before the updated cluster is running with new roles, otherwise the running cluster might start to fail.

Reply:
as we are adding the CSI roles to the create.yaml which will be called with `kubectl create` will the new rules be applied? 

the roles name are same but it contains extra permission in v1.1, deleting these roles after the upgrade is done can cause the cluster to go unhealthy  and people who has not deployed CSI in v1.0 need to create all the roles